libpod/networking.go

```diff
@@ -102,3 +108,63 @@ func (r *Runtime) teardownNetNS(ctr *Container) error {
 
 	return nil
 }
+
+//ReadPortRangeFile reads the range of available ports per proc
+func ReadPortRangeFile() (int, int, error) {
+	portFileName := "/proc/sys/net/ipv4/ip_local_port_range"
+	portFile, err := ioutil.ReadFile(portFileName)
+	if err != nil {
+		return 0, 0, errors.Wrapf(err, "unable to read %s", portFileName)
+	}
+	portFields := bytes.Fields(portFile)
+	if len(portFields) < 2 {
+		return 0, 0, fmt.Errorf("invalid data in %s", portFileName)
+	}
+	begin, err := strconv.Atoi(string(portFields[0]))
+	if err != nil {
+		return 0, 0, errors.Wrapf(err, "unable to convert port range data to int")
+	}
+
+	end, err := strconv.Atoi(string(portFields[1]))
+	if err != nil {
+		return 0, 0, errors.Wrapf(err, "unable to convert port range data to int")
+	}
+
+	return begin, end, nil
+}
+
+//GetRandomPort returns a unused port given the port range as defined in /proc
+func GetRandomPort() (int, error) {
```

Comment:
Move this out to cmd/kpod, unlikely to be used in libpod proper

Reply:
done